### PR TITLE
LESQ 1042 & LESQ-1038 - clerk sign out button and form styles

### PR DESCRIPTION
**Pull request by JBR90**
## Description

- Replaces sign out in hamburger with clerk signout in app header
- Updates clerk style to match OWA (as much as possible considering limitations) 

## Acceptance Criteria:

- [ ]  Use the default style clerk sign out option (what AI squad currently has)
- [ ]  Should only be displayed if you are logged in with the feature flag enabed
- [ ] Logging out via the component should have the same behaviour as logging out currently does (logging the user out and redirecting them to the homepage)
- [ ]  remove log out button from hamburger

- [ ] Make sure sign in and sign up match Figma in terms of Font sizes, colours, spacing, border radius etc
- [ ]  Make name fields required

https://www.figma.com/design/fxWHjjNzvbbc0J38pM3Kti/Log-in?node-id=2343-5313&node-type=SECTION&m=dev

[[Ticket checklist (non-tester)](https://www.notion.so/Ticket-checklist-non-tester-5317535736fe47fb984a3591a1bd7616?pvs=21)](https://www.notion.so/Ticket-checklist-non-tester-5317535736fe47fb984a3591a1bd7616?pvs=21)

## Issue(s)

Fixes #

## How to test

1. Go to https://deploy-preview-2757--oak-web-application.netlify.thenational.academy
Go to lesson download and sign in, you should see avatar in app header, test criteria above. 

## Checklist

- [ ] Added or updated tests where appropriate
- [ ] Manually tested across browsers / devices
- [ ] Considered impact on accessibility
- [ ] Design sign-off
- [ ] Approved by product owner
- [ ] Does this PR update a package with a breaking change

### CODE REVIEW

**Comment by JBR90** [comment]
@Monika-Koziol The logo looks a bit pixilated to be me, do you have another version I could use ?

File: src/app/(registration)/formAppearanceStyles.ts [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+      fontSize: "20px",
+      textAlign: "start",
+      marginBottom: "8px",
+      color: "#222222",
+      fontWeight: "600",
+    },
+    headerSubtitle: {
+      fontSize: "16px",
+      textAlign: "start",
+      color: "#575757",
```
Comment (njblore) [inline src/app/(registration)/formAppearanceStyles.ts:20]: thinking about it we might want to use a fn to get these colours by name so if they change hex codes we don't need to update them here

**Review by njblore — APPROVED** [review]
Looking good!

**Comment by RBrownsett** [comment]
@JBR90 looking good - a couple observations:

When in mobile view and the button is expanded the content on the screen is pushed over to the left a little:

image

When navigating between screens it can take a few moments for the sign out button to appear, not sure if this can be improved? Its not too bad, just a little noticeable: 

https://github.com/user-attachments/assets/97701c08-434c-4089-b42a-1674428f9878

Maybe more of a design question @Monika-Koziol (or not sure if we are limited with what we can do) when the account button is in focus the highlighted area is difficult to see with the current colour scheme. 

image

**Comment by Monika-Koziol** [comment]
@JBR90 is this acorn better? We use that as a favicon and it has straight lines. 
Acorn
In terms of focus style. Can we style that? If so can we match Oak style box-shadow: 0px 0px 0px 2px #FFE555, 0px 0px 0px 5px #575757; also in storybook: https://components.thenational.academy/?path=/story/styletokens-oakdropshadowtokens--double-shadow

**Comment by JBR90** [comment]
> https://www.figma.com/design/fxWHjjNzvbbc0J38pM3Kti/Log-in?node-id=2343-5313&node-type=SECTION&m=dev



> @JBR90 is this acorn better? We use that as a favicon and it has straight lines. 

@Monika-Koziol - Do you have a PNG of this ?


 In terms of focus style. Can we style that? If so can we match Oak style box-shadow: 0px 0px 0px 2px #FFE555, 0px 0px 0px 5px #575757; also in storybook: https://components.thenational.academy/?path=/story/styletokens-oakdropshadowtokens--double-shadow
 
 Yes just updated the box-border on last commit

**Comment by JBR90** [comment]
> @JBR90 looking good - a couple observations:
> 
> When in mobile view and the button is expanded the content on the screen is pushed over to the left a little:
> 
This should be fixed now.

> When navigating between screens it can take a few moments for the sign out button to appear, not sure if this can be improved? Its not too bad, just a little noticeable:

Not found a way to improve this i'm afraid, limited because it's a clerk component.
> 
>  Screen.Recording.2024-09-09.at.09.39.43.mov 
> Maybe more of a design question @Monika-Koziol (or not sure if we are limited with what we can do) when the account button is in focus the highlighted area is difficult to see with the current colour scheme.
> 

Have changed to the oak box shadow - is the contract ok in the drop down items ?

**Review by RBrownsett — APPROVED** [review]
Approved - profile button now how has focus, clicking on profile in mobile view does not affect the page layout. 

Sign out button removed from hamburger menu
Sign out flow remains unaffected
Does not show when feature flag is not enabled

**Pull request merged**